lua/mason-lspconfig/mappings/filetype.lua

```diff
@@ -1,208 +1,209 @@
+    ql = { "codeqlls" },
+    quarto = { "ltex" },
+    r = { "r_language_server" },
+    raku = { "raku_navigator" },
+    razor = { "tailwindcss" },
+    reason = { "ocamllsp", "reason_ls", "tailwindcss" },
+    rescript = { "rescriptls", "tailwindcss", "unocss" },
+    rmd = { "ltex", "r_language_server" },
+    rnoweb = { "ltex" },
+    robot = { "robotframework_ls" },
+    rst = { "esbonio", "ltex" },
+    ruby = { "rubocop", "ruby_ls", "solargraph", "sorbet", "standardrb" },
+    rust = { "ast_grep", "rust_analyzer" },
+    sass = { "emmet_language_server", "emmet_ls", "tailwindcss" },
+    scss = { "cssls", "emmet_language_server", "emmet_ls", "stylelint_lsp", "tailwindcss" },
+    sh = { "bashls" },
+    slim = { "tailwindcss" },
+    slint = { "slint_lsp" },
+    sls = { "salt_ls" },
+    smithy = { "smithy_ls" },
+    solidity = { "solang", "solc", "solidity", "solidity_ls_nomicfoundation" },
+    sql = { "sqlls", "sqls" },
+    stylus = { "tailwindcss" },
+    sugarss = { "stylelint_lsp", "tailwindcss" },
+    surface = { "elixirls" },
+    svelte = { "emmet_language_server", "emmet_ls", "eslint", "svelte", "tailwindcss", "unocss" },
+    svg = { "lemminx" },
+    systemverilog = { "svlangserver", "svls", "verible" },
+    teal = { "teal_ls" },
+    templ = { "templ" },
+    terraform = { "terraformls", "tflint" },
+    ["terraform-vars"] = { "terraformls" },
+    tex = { "ltex", "texlab" },
+    text = { "vale_ls" },
+    thrift = { "thriftls" },
+    toml = { "taplo" },
+    twig = { "tailwindcss" },
+    typescript = { "angularls", "ast_grep", "biome", "cssmodules_ls", "denols", "ember", "eslint", "glint", "rome", "sourcery", "stylelint_lsp", "tailwindcss", "tsserver", "vtsls" },
+    ["typescript.glimmer"] = { "ember", "glint" },
+    ["typescript.tsx"] = { "angularls", "biome", "denols", "eslint", "rome", "tsserver", "vtsls" },
+    typescriptreact = { "angularls", "biome", "cssmodules_ls", "denols", "emmet_language_server", "emmet_ls", "eslint", "graphql", "rome", "sourcery", "stylelint_lsp", "tailwindcss", "tsserver", "unocss", "vtsls" },
+    typst = { "typst_lsp" },
+    v = { "vls" },
+    vala = { "vala_ls" },
+    vb = { "omnisharp", "omnisharp_mono" },
+    verilog = { "svlangserver", "svls", "verible" },
+    veryl = { "veryl_ls" },
+    vim = { "vimls" },
+    visualforce = { "visualforce_ls" },
+    vlang = { "vls" },
+    vmasm = { "asm_lsp" },
+    vue = { "emmet_language_server", "emmet_ls", "eslint", "stylelint_lsp", "tailwindcss", "unocss", "volar", "vuels" },
+    wgsl = { "wgsl_analyzer" },
+    wxss = { "stylelint_lsp" },
+    xml = { "lemminx" },
+    xsd = { "lemminx" },
+    xsl = { "lemminx" },
+    xslt = { "lemminx" },
+    yaml = { "azure_pipelines_ls", "spectral", "yamlls" },
+    ["yaml.ansible"] = { "ansiblels" },
+    ["yaml.docker-compose"] = { "docker_compose_language_service", "yamlls" },
+    yml = { "spectral" },
+    zig = { "zls" },
+    zir = { "zls" }
+}
```

Comment:
the formatting changes might become a blocker.

Reply:
oh, ok, sorry, I'll fix that!